lib/features/guide/widgets/about_us/widgets/desription_section.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_widget_from_html_core/flutter_widget_from_html_core.dart';
+
+import '../../../../../config/ui_config.dart';
+import '../../../../../theme/app_theme.dart';
+import '../../../../../utils/launch_url_util.dart';
+
+class DescriptionSection extends StatelessWidget {
+  const DescriptionSection({super.key, required this.text});
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding:
+          const EdgeInsets.symmetric(horizontal: AboutUsConfig.defaultPadding),
+      child: Container(
+        width: double.infinity,
+        decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular(AboutUsConfig.borderRadius),
+          color: context.colorTheme.greyLight,
+        ),
+        child: Padding(
+          padding: const EdgeInsets.all(AboutUsConfig.defaultPadding),
+          child: HtmlWidget(
+            text,
+            textStyle: context.aboutUsTheme.body,
+            customStylesBuilder: (element) {
+              if (element.localName == 'a') {
```

Comment:
I don't like this `'a'` magic string

but I see it's hard to avoid this with this `HtmlWidget` :/

the best I can come up with is maybe smth similar to this in separate file:
```dart
import 'package:flutter/material.dart';
import 'package:html/dom.dart' as html;

import '../../../../../theme/app_theme.dart';
import '../../../../../theme/hex_color.dart';

extension ToHtlColorString on HexColor {
  String get htmlFormat =>
      '#${value.toRadixString(16).padLeft(8, '0').substring(2)}';
}

extension IsLinkTag on html.Element {
  bool get isLink {
    return localName == "a";
  }
}

extension CustomHtmlStyles on BuildContext {
  Map<String, String>? customStylesBuilder(html.Element element) {
    if (element.isLink) {
      return {
        'color': colorTheme.orangePomegranade.htmlFormat,
        'text-decoration': 'none',
      };
    }
    return null;
  }
}
```